io.home-assistant.fan/manifest.tt

```diff
@@ -0,0 +1,33 @@
+abstract class @io.home-assistant.fan
+#_[thingpedia_name="Fan"]
+#_[thingpedia_description="Interface for Home Assistant's Fan."]
+#_[description="Interface for Home Assistant's Fan."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(on,off))
+  #_[canonical="fan state"]
+  #_[confirmation="the state of your fan"]
+  #_[formatted=["Your fan is ${state}"]];
+
+  action turn_on()
+  #_[canonical="turn on fan"]
+  #_[confirmation="turn on your fan"]
+  #[confirm=true];
+
+  action turn_off()
+  #_[canonical="turn off fan"]
+  #_[confirmation="turn off your fan"]
+  #[confirm=true];
```

Comment:
Most often, it's preferrable to merge related functions with similar utterances into one function with enum parameters, because the network learns to better align the function part ("turn", "fan") and the parameter part ("on" / "off").
I understand for start/stop or open/close it's somewhat hard to write in a manner that makes the templates and confirmations natural, but for on/off we should definitely merge.

Reply:
Alright, I'll merge all the turn_on/turn_off actions